README.md

```diff
@@ -14,21 +16,21 @@ public RequestTraceFilter requestTraceFilter(TraceRepository traceRepository, Tr
 ```
 
 ## @APIDeprecated
-Added an annotation named `@APIDeprecated` for classes and methods that add warning headers to the responses generated by the API.
+An annotation for controller classes and methods that adds warning headers to the responses generated by the API.
 
-A typical usage would be to add the @APIDeprecated annotation to a method as in the following example.
+A typical usage would be to add the `@APIDeprecated` annotation to a method as in the following example:
 
-```
-    @RequestMapping("/deprecated")
-    @APIDeprecated(
-            name = "Deprecated Endpoint",
-            expiryDate = "01/06/2018",
-            docLink = "https://example.org/docs/api/deprecated/",
-            note = "Just a note, upgrade your client code!!!"
-    )
-    public String deprecated() {
-        return "Greetings from deprecated Class Controller!";
-    }
+```java
+@RequestMapping("/deprecated")
+@APIDeprecated(
+    name = "Deprecated Endpoint",
+    expiryDate = "2018-06-30",
+    docLink = "https://example.org/docs/foo",
```

Comment:
why this change?

Reply:
Didn't want to suggest that the docLink url matches actual deprecated route